### ISSUE TITLE
Accessibility - Display Issue When VoiceOver Reads Image after Search Button

### ISSUE THREAD

**Issue by jfc3-dol** [USDepartmentofLabor/Ilab-do-good#20]
There is a display issue that occurs when VoiceOver reads I think the back ground image after the search button. The whole application moves down by a third of the screen. I think VoiceOver is saying "task" or "test" JPG image.

Not sure what or where the image is. My guess it's the background image but I could be wrong.

Will add screenshot when I have time to show what is happening.

**Comment by adriennerlong** [comment]
I'm guessing this is referring to the image of the girl at the sewing machine? Do we need to have a voice over providing a description of the photo? If so, I'll mark this as a high priority issue and ask developers to address before 9/11.

**Comment by jfc3-dol** [comment]
@adriennerlong,

If it is a background image with CSS no ALT text is required. I'm not sure without looking at the code what item VoiceOver is reading and why it's moving the screen down.

**Comment by jfc3-dol** [comment]
Not closing this issues because it's has not been fixed. The image is moved down screen after reading search icon and read the next item which is background image.

Attached is a screenshot to show what it looks like.

I'm using our iPad Air 2 with iOS 10.3.3 operating system.

**Comment by adriennerlong** [comment]
Hi John, could you explain what CSS is? Also, I don't see a screen shot so I'm not sure how to advise ICF to fix this issue. Below are screenshots for the landing page before the search function is enabled and after the search function is enabled. It appears that when the search function is enabled it moves the top 3 menu items down. 
![landing page](https://user-images.githubusercontent.com/31417858/29974991-c1bb3ece-8f02-11e7-8e14-987672a01a95.png)
![search function](https://user-images.githubusercontent.com/31417858/29974990-c1b10062-8f02-11e7-866c-2262621cf226.png)

**Comment by jfc3-dol** [comment]
@adriennerlong,

CSS is what styles the look of the page/screen. Sometimes you can make the image part of the HTML code or a background image that is not seen by a screen reader user and does not need ALT text.

Left me find the screenshot I took. 

The red text is my added note to help me remember and tell others of the issue. The red text is where  the image is pushed down. The problem happens when VoiceOver reads the serach button then the background image of the girl then corrects itself when reading the information icon.

![complychain - main screen - images is moved down screen after reading search icon and read the next item which is background image](https://user-images.githubusercontent.com/10423487/29984426-6c49a392-8f28-11e7-95ea-57bc40d62fec.png)

**Comment by jfc3-dol** [comment]
@adriennerlong,

Here is image without red text. Added dashed gray line so you can see how much of a gap there is.

-----------------------------------------------------------------------------
![img_0152](https://user-images.githubusercontent.com/10423487/29984539-e6c81cde-8f28-11e7-8d8e-426d27e170a0.PNG)

**Comment by adriennerlong** [comment]
Is there any issue in making the photo of the girl a background image that is not seen by a screen reader user? I wonder if that will help address the issue.

**Comment by jfc3-dol** [comment]
@adriennerlong,

My guess making the image background image should correct the issue but will not know until it's changed and I test it.

**Comment by jfc3-dol** [comment]
@adriennerlong,

Background image no longer moves down the screen when using VoiceOver to read through the screen.

Closing isssue.
